components/dropdown/test/dropdown-menu.visual-diff.js

```diff
@@ -0,0 +1,27 @@
+const puppeteer = require('puppeteer');
+const VisualDiff = require('@brightspace-ui/visual-diff');
+const helper = require('./dropdown-helper.js');
+
+describe('d2l-dropdown-menu', () => {
+
+	const visualDiff = new VisualDiff('dropdown-menu', __dirname);
+
+	let browser, page;
+
+	before(async() => {
+		browser = await puppeteer.launch();
+		page = await browser.newPage();
+		await page.setViewport({ width: 800, height: 800, deviceScaleFactor: 2 });
+		await page.goto(`${visualDiff.getBaseUrl()}/components/dropdown/test/dropdown-menu.visual-diff.html`, { waitUntil: ['networkidle0', 'load'] });
+		await page.bringToFront();
+	});
+
+	after(() => browser.close());
+
+	it('first-page', async function() {

```

Comment:
I'm planning on adding an additional case for the second page once the `hierarchical-view` issue is fixed to ensure that the `dropdown-menu` content resizes properly when the inner menu resizes. Right now it can't be added without accessing `d2l-menu` / `d2l-hierarchical-view` implementation details or a hard-coded timeout to determine when the animation ends.